Tests/SwiftSyntaxTest/IdentifierTests.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(RawSyntax) import SwiftSyntax
+import XCTest
+
+class IdentifierTests: XCTestCase {
+    public func testIdentifierInit() {
+        let someToken = TokenSyntax(stringLiteral: "someToken")
+        XCTAssertNotNil(Identifier(someToken))
+        
+        let nonIdentifierToken = DeclSyntax("let a = 1").firstToken(viewMode: .all)!
```

Comment:
Could you use `XCTUnwrap` instead of the force unwrapping? That way test execution continues and doesn’t crash if `firstToken` should be `nil`.